src/main/java/com/redhat/cloud/notifications/models/Endpoint.java

```diff
@@ -15,12 +15,14 @@
 
 public class Endpoint {
 
-    @Schema(enumeration = { "webhook", "email", "default" })
+    @Schema(enumeration = { "webhook", "email", "email_subscription", "default" })
     public enum EndpointType {
         @JsonProperty("webhook")
         WEBHOOK,
         @JsonProperty("email")
         EMAIL,
+        @JsonProperty("email_subscription")
```

Comment:
What is the difference to 'email'?

Reply:
`email` is to send direct emails (or it was implemented in such way) instead of querying who is subscribed to the emaisl. I think we could remove it for now and put it back if needed.